dev_requirements.txt

```diff
@@ -9,13 +9,15 @@ attrs==20.2.0             # via flake8-bugbear, pytest, syrupy
 black==20.8b1             # via syrupy
 bleach==3.2.1             # via readme-renderer
 certifi==2020.6.20        # via requests
+cffi==1.14.3              # via cryptography
 chardet==3.0.4            # via requests
 click==7.1.2              # via black, pip-tools
-codecov==2.1.10            # via syrupy
+codecov==2.1.10           # via syrupy
 colorama==0.4.4           # via twine
 colored==1.4.2            # via syrupy
 configparser==5.0.1       # via py-githooks
 coverage[toml]==5.3       # via codecov, syrupy
+cryptography==3.2.1       # via secretstorage
```

Comment:
This comes from secretstorage which is a direct dependency of keyring. It's only required on linux and so disappears whenever requirements are updated manually on Mac.